README.md

```diff
@@ -10,7 +10,18 @@ Install the necessary packages by using the command after you have installed yar
 
 `yarn dev`
 
-If you receive an error about GitHub permission being denied, delete the file 'yarn-lock.json', then run the command again.
+Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init), [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/), or [pnpm](https://pnpm.io) to bootstrap the example:
+
+```bash
+npx create-next-app --example with-tailwindcss with-tailwindcss-app
```

Comment:
we do we need these instructions?

Reply:
For readme I think I merged main before the new main and that's why it is showing those next app lines. When we will update this branch with the new main I will remove this